include/items.php

```diff
@@ -422,7 +422,7 @@ function uri_to_guid($uri, $host = "") {
  * @return array Item array with removed conversation data
  */
 function store_conversation($arr) {
-	if (in_array($arr['network'], array(NETWORK_DFRN, NETWORK_DIASPORA, NETWORK_OSTATUS)) && !empty($arr['uri'])) {
+	if (in_array(defaults($arr, 'network', NETWORK_PHANTOM), array(NETWORK_DFRN, NETWORK_DIASPORA, NETWORK_OSTATUS)) && !empty($arr['uri'])) {
```

Comment:
Wouldn't it be better to define a variable like ```$network``` at the beginning of the function instead of having to cope with a possible undefined array value at every location in the function?

Reply:
Changes added in `item_store` where I used `defaults()` twice.